gordon_gcp/schemas/event.schema.json

```diff
@@ -1,3 +1,49 @@
 {
-    "foo": "bar"
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/event.schema.json",
+    "title": "Generic Event Messages",
```

Comment:
The idea for this second schema is for messages that are not automatically produced by Google's Audit Log of operations (i.e. manual records, a reconciliation process, etc).